transports/transport-rest/transport-rest-vertx/src/main/resources/config/base/log4j.properties

```diff
@@ -0,0 +1,13 @@
+# access log default configuration
```

Comment:
The log configure should be provide from the Application level

Reply:
User can override the access log configuration in their log4j.properties. Or totally replace Log4j with other log printer framework, since this AccessLogHandlerImpl just depends on slf4j.